rfc#0001

```diff
@@ -0,0 +1,140 @@
+# WASM binary loading into an Enarx Keep
```

Comment:
I recently learned that *Wasm* is the correct abbreviation of WebAssembly:
> A contraction of “WebAssembly”, not an acronym, hence not using all-caps. 

--  [WebAssembly draft spec](https://webassembly.github.io/spec/core/bikeshed/index.html#id1)

Reply:
Changing throughout: thanks.